src/perform_step/rosenbrock_perform_step.jl

```diff
@@ -1449,6 +1453,38 @@ end
     cache.linsolve = linres.cache
 end
 
+function calculate_interpoldiff(uprev, up2, up3, c_koeff, d_koeff, c2_koeff)
+    u_int = 0.0
+    u_diff = 0.0
+    for i in eachindex(up2)
```

Comment:
this will have issues with GPU-ability.

Reply:
The for loop can be omitted because only scalar problems are handled here.